scripts/linux_share_build.sh

```diff
@@ -3,33 +3,68 @@ PROC=$((`grep -c processor /proc/cpuinfo`))
 
 function install_prerequisites() {
     TZ=America/Chicago
-    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
-    apt-get update -y
-    apt-get install -y g++ libeigen3-dev libhdf5-dev patchelf git cmake
+    $SUDO ln -snf /usr/share/zoneinfo/$TZ /etc/localtime
+    $SUDO sh -c 'echo $TZ > /etc/timezone'
+    $SUDO apt-get update -y
+    $SUDO apt-get install -y g++ libeigen3-dev patchelf git cmake curl
 }
 
-function setup_folder() {
+function setup() {
+    unset LD_LIBRARY_PATH
+    
+    echo "Building the Trelis plugin in ${CURRENT}\\${PLUGIN_DIR}"
     cd ${CURRENT}
     mkdir ${PLUGIN_DIR}
-    echo "Building the Trelis plugin in ${CURRENT}\\${PLUGIN_DIR}"
-
     cd ${PLUGIN_DIR}
     PLUGIN_ABS_PATH=$(pwd)
-    ln -s $SCRIPTPATH/ ./
+    ln -s ${SCRIPTPATH}/ ./
+}
+
+function setup_var() {
+    # Setup the variables
+    if [ "$1" = "2020.2" ]; then
+        TRELIS_PATH="/opt/Coreform-Cubit-2020.2"
+        TRELIS_PKG="Coreform-Cubit-2020.2-Lin64.deb"
+        TRELIS_SDK_PKG="Coreform-Cubit-2020.2-Lin64-SDK.tar.gz"
+    elif [ "$1" = "17.1.0" ]; then
+        TRELIS_PATH="/opt/Trelis-17.1"
+        TRELIS_PKG="Trelis-17.1.0-Lin64.deb"
+        TRELIS_SDK_PKG="Trelis-SDK-17.1.0-Lin64.tar.gz"
+        CMAKE_ADDITIONAL_FLAGS="-DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0"
+    else
+        echo "unknown Trelis/Cubit version, use: \"17.1\" or \"2020.2\""
```

Comment:
If this is in fact an error, let's return a non-zero exit code for the function to indicate the failure and stop the script.

```suggestion
        echo "unknown Trelis/Cubit version, use: \"17.1\" or \"2020.2\""
        return 1
        
```